R2API/DotAPI.cs

```diff
@@ -105,12 +111,12 @@ public static class DotAPI {
         /// <param name="interval"></param>
         /// <param name="damageCoefficient"></param>
         /// <param name="colorIndex"></param>
-        /// <param name="associatedBuff"></param>
+        /// <param name="associatedBuff">The buff associated with the DOT, can be null</param>
         /// <param name="customDotBehaviour"></param>
         /// <param name="customDotVisual"></param>
         /// <returns></returns>
         public static DotController.DotIndex RegisterDotDef(float interval, float damageCoefficient,
-            DamageColorIndex colorIndex, BuffDef associatedBuff, CustomDotBehaviour customDotBehaviour = null,
+            DamageColorIndex colorIndex, BuffDef associatedBuff = null, CustomDotBehaviour customDotBehaviour = null,
```

Comment:
that kills retro compatibility no ?

Reply:
It shouldn't kill retro-compatibility at all i think. considering that you can just overwrite the default value of the dot def

Reply:
Oh yeah, probably

Reply:
Tested out ingame with Starstorm2Nightly, which is compiled to use the newest version of r2api, there where no errors regarding both of it's DOT's (Gouge and Trematodes)